erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -52,6 +52,13 @@ def on_submit(self):
 		self.update_loan()
 		self.update_inter_company_jv()
 
+	def before_print(self):
+		gl_entries = frappe.get_list("GL Entry",filters={"voucher_type": "Journal Entry",
+			"voucher_no": self.name} ,
+			fields=["account", "party_type", "party", "debit", "credit", "remarks"]
+		)
+		self.gl = gl_entries
```

Comment:
No need of gl_entries variable, assign the value of frappe.get_list directly to self.gl_entries instead of self.gl

Reply:
done